keyboards/kinesis/keymaps/TuesdayJohn/readme.md

```diff
@@ -0,0 +1,246 @@
+
+There are minor changes in the base keywell layout of non-alpha keys (e.g., CapsLock is ESC/Ctl, arrow clusters on one side), while the thumb clusters deviate more from the default layout.
+
+Changes to the thumb clusters include: 
+* The navigations keys moved to function layers.
+* The function of 2u keys have been reversed - I've always used space with my left thumb, and I find it more helpful to have quick and easy access to Space and Enter while using my mouse/trackball.
+* Backspace have been duplicated on both clusters.  As with Space and Backspace, I find it helpful to have quick and easy access to Backspace while using my mouse/trackball.
+* The 2u keys serve dual function as momentary layer switchers.
+
+I've largely left the function keys untouched, with the intension of not using them often.  They are neither easy to use nor reach due to their locations and size, and Kinesis used not-so-great rubber dome switches for them (Advantage model and earlier).  Kinesis have since replaced the keys with Cherry ML switches on Advantage2, but did not resolve the issues of size or location.  Additionally, leaving the function keys unused here makes it easier for me to adapt the keymaps to my other keyboards, most of which do not have physical function keys.
+
+I use Colemak as my default layout.  I've included QWERTY here as well.
+
+Additionally, I've added gaming/vanilla version of Colemak and QWERTY layouts.  These layouts have limited access to the Function or Numbers/Symbols layers, and are mainly used for gaming.  Additionally, NKRO is turned on by default for these layers.
+
+Persistent default layer has been enabled for Colemak and QWERTY.  The gaming/vanilla Colemak and QWERTY can be set as default layer, but will not be persistent.
+
+## Future plans
+
+* Add status LEDs to the Stapelberg PCB (usually used for Caps Lock, Num Lock, and Scroll Lock) to use as layer indicators.
+* Add a speaker now that QMK supports additional pins for audio use.
+* Utilize the leftover spots on the key matrix, as well as unused pins on Teensy++ 2.0 to run macropad and/or foot pedals.
+
+
+## Layers
+
+### Function Keys on all layers
+- 'Numpd' toggles the Numpad layer
+- 'Adjst' is a momentary layer key to access the Adjust layer
+
+```
+,-----------------------------------------------------. ,----------------------------------------------------.
+| ESC |  F1 |  F2 |  F3 |  F4 |  F5 |  F6 |  F7 |  F8 | | F9 | F10 | F11 | F12 | PScr| SLck| Paus|Numpd|Adjst|
+`-----------------------------------------------------' `----------------------------------------------------'
+```
+
+### Colemak
+- Default layer
+- Keys separated by "/" tap for first, hold for second
+- Uses [Space Cadet Shifts](https://beta.docs.qmk.fm/features/feature_space_cadet_shift)
```

Comment:
The page name is actually `feature_space_cadet`; your current link is Error 404.

```suggestion
- Uses [Space Cadet Shifts](https://beta.docs.qmk.fm/features/feature_space_cadet)
```